modules/webgl/src/adapter/webgl-device.ts

```diff
@@ -297,12 +298,16 @@ ${device.info.vendor}, ${device.info.renderer} for canvas: ${device.canvasContex
     return new WEBGLFramebuffer(this, props);
   }
 
-  createRenderPipeline(props: RenderPipelineProps): WEBGLRenderPipeline {
-    return new WEBGLRenderPipeline(this, props);
+  createVertexArray(props: VertexArrayProps): VertexArray {
+    return new WEBGLVertexArray(this, props);
   }
 
-  override createVertexArray(props: VertexArrayProps): VertexArray {
```

Comment:
Is it insignificant to drop `override` here?

Reply:
Depends on whether this is an abstract method etc. Confusingly I can't see a change in the base class that requires it, not sure why it came in, but it passed tsc and linter so should be fine.